R/users.R

```diff
@@ -179,3 +179,23 @@ users_exist <- function(vault, users) {
 on_failure(users_exist) <- function(call, env) {
   paste0("Secret ", deparse(call$users), " do not exist")
 }
+
+#' @importFrom openssl fingerprint read_pubkey
+
+lookup_user <- function(key, vault = NULL) {
```

Comment:
No default value needed for `vault`. Also no `find_vault` call is needed. The public API functions find the vault already.